EnglishFilter/sections/adservers.txt

```diff
@@ -5,6 +5,7 @@
 !
 !! https://a.bestcontentitem.top/warp/*?r=
 !
+||activehosted.com^$third-party
```

Comment:
no. This is annoyance. Please move to this section:
https://github.com/AdguardTeam/AdguardFilters/blob/d0084bf95ad0616a03e927456c51d25d660134a6/AnnoyancesFilter/sections/annoyances.txt#L1760